frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx

```diff
@@ -520,7 +519,7 @@ export const DEPRECATED_CreateOperandForm: React.FC<OperandFormProps> = ({
       const existing = immutableFormData.getIn([...pathToArray(pathBeforeIndex), 0]);
       const item = Immutable.Map(existing || {}).setIn(pathToArray(pathAfterIndex), value);
       const list = Immutable.List([item]);
-      onChange(immutableFormData.setIn(pathToArray(pathBeforeIndex), list));
+      onChange(immutableFormData.setIn(pathToArray(pathBeforeIndex), list).toJS());
```

Comment:
This is not technically part of the bug fix, but was a problem that I noticed while scanning the code. onChange shouldn't return an Immutable.Map instance.